Makefile.in

```diff
@@ -288,13 +303,13 @@ fuzzstandalone: FUZZLIB=fuzz/fuzz-harness.o
 fuzzstandalone: fuzz/fuzz-harness.o fuzz-targets
 
 # Build all the fuzzers. Usually like
-#   make fuzz-targets FUZZLIB=-lFuzzer.a 
+#   make fuzz-targets FUZZLIB=-lFuzzer.a
 # the library provides main(). Otherwise
 #   make fuzzstandalone
 # provides a main in fuzz-harness.c
 fuzz-targets: $(FUZZ_TARGETS) $(FUZZER_OPTIONS)
 
-$(FUZZ_TARGETS): $(FUZZ_OBJS) $(allobjs) $(LIBTOM_DEPS) 
+$(FUZZ_TARGETS): $(FUZZ_OBJS) $(allobjs) $(LIBTOM_DEPS)
```

Comment:
Perhaps adding `$(HEADERS)` as a dependency here might fix the latest cifuzz failure? Otherwise it won't generate `default_options_guard.h` - not sure how it worked before

Reply:
Latest commit was able to fix that. This was missing previously:
```
Creating default_options_guard.h
```
https://github.com/mkj/dropbear/actions/runs/4094539903/jobs/7060822066#step:4:579

I ended up reordering the dependencies.
Nevertheless it wasn't enough. Still something's amiss...

tjk :/